setup.sh

```diff
@@ -29,6 +29,10 @@
 SKIP_TRIGGERS=${SKIP_TRIGGERS:-}
 SKIP_AUTH=${SKIP_AUTH:-}
 
+# Cleanup residual Terraform resources
+export REGION="us-central1"
+sh ./terraform/cleanup.sh
```

Comment:
We shouldn't include destroy/teardown in this script, we should assume a clean state or the state the developer wants. We can have another script that does teardown first.

Reply:
I've moved that to `terraform/cleanup.sh`.